packages/fxa-auth-server/lib/customs.js

```diff
@@ -151,9 +149,35 @@ module.exports = function(log, error, statsd) {
       query: request.query,
       payload: sanitizePayload(request.payload),
     });
+
+    optionallyReportStatsD('request.check', action, result);
+
     return handleCustomsResult(request, result);
   };
 
+  function optionallyReportStatsD(name, action, options = {}) {
+    if (statsd) {
+      const { block, unblock, suspect, blockReason } = options;
+
+      // The action and block values will always be reported, all others are optional
+      const tags = { action, block };
+
+      if (options.unblock !== undefined) {
+        tags['unblock'] = unblock;
+      }
+
+      if (options.suspect !== undefined) {
+        tags['suspect'] = suspect;
+      }
+
+      if (options.blockReason !== undefined) {
+        tags['blockReason'] = blockReason;
+      }
+
+      statsd.increment([serviceName, name].join('.'), tags);
```

Comment:
@chenba Any objections to doing statsd reporting like this or anything I could optimize?

Reply:
That LGTM.  As long as the potential set of values for the tags are pretty limited.  I believe they are, but I don't know the customs server well enough to say.

An aside: not sure which is better, array `join` or a template literal.